### PR TITLE
Add a Candlestick charts section (#285)

### LINKED ISSUE #285: Create a candlestick charts section
**Description**

See [candlestick charts content](https://docs.google.com/document/d/1v4x69_bBsc1B5dBu_8I05xtDdUGI0nTa).

### PR DESCRIPTION
**Pull request by drake-stamos**
Addresses #285.

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/candlestick-charts.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+## Candlestick Chart Key Parameters <a id="table"></a>
+Explanation of parameters and options relevant to candlestick charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
+
+| Parameter       | Value Options             | Location           |
+| --------------- | ------------------------- | ------------------ |
+| Type          | "candlestick"          | Control definition |
+| high,low,open,close | Values to plot | Control definition |
+| rangeslider | dict of parameters in axis layout | layout |
+
+## Required Libraries
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/candlestick-charts.md]: Needs to indicate that this is the required libraries for the demo below, not a required libraries for candlestick charts to work

File: docs/manuals/gui/viselements/charts/candlestick-charts.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+## Candlestick Chart Key Parameters <a id="table"></a>
+Explanation of parameters and options relevant to candlestick charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
+
+| Parameter       | Value Options             | Location           |
+| --------------- | ------------------------- | ------------------ |
+| Type          | "candlestick"          | Control definition |
+| high,low,open,close | Values to plot | Control definition |
+| rangeslider | dict of parameters in axis layout | layout |
+
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/candlestick-charts.md]: Should indicate that this is the start of the demos/examples

File: docs/manuals/gui/viselements/charts/candlestick-charts.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+## Required Libraries
+- [yfinance](https://pypi.org/project/yfinance/)
+  
+```sh
+pipenv install yfinance
+```
+
+## Simple Candlestick with Pandas
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/candlestick-charts.md]: More details on what this candlestick charts is about (ex: Candlestick on apple charts for the last 50 years (1 year interval))

**Review by FredLL-Avaiga — APPROVED** [review]
No comment 🕯️

**Review by dinhlongviolin1 — APPROVED** [review]
Ready to go

**Pull request merged**